client/src/Nav.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import './App.css';
+import { Link } from 'react-router-dom';
+
+function Nav(props) {
+
+  return (
+    <nav className="top-nav">
+      <a className="link-nav" href="http://localhost:3000/"><h3>YOSO</h3></a>
+      <div className="profile-comp">
+      <img src="https://cataas.com/cat/says/hello%20world!" className="img-avatar" alt="cat"></img>
```

Comment:
No kittens anymore. There's `photo` property returned by `GET /users/current/profile`, use that instead.